docs/src/user_docs/assembly/code_organization.md

```diff
@@ -170,46 +170,46 @@ In the module shown above, not only is the locally-defined procedure `foo` expor
 Similar to procedure invocation, you can bypass the explicit import by specifying an absolute path, like so:
 
 ```
-export.::std::math::u64::mul->mul64
+pub use ::std::math::u64::mul->mul64
 ```
 
-Additionally, you may re-export a procedure using its MAST root, so long as you specify an alias:
+Additionally, you may re-export a procedure using its MAST root, so long as you specify a name for it, as shown below:
 
 ```
-export.0x0000..0000->mul64
+pub use 0x0000..0000->mul64
```

Comment:
Question: how does this get resolved? I'm assuming that the compilation will fail if the assembler cannot find a procedure with such root during compilation - right? Either way, probably worth briefly explaining this.

Reply:
It won't be rejected, since it is a MAST root, instead, any use of the `mul64` symbol will be treated as if the aliased MAST root had been used directly, and the assembler presumes that any explicit MAST root reference like this will be resolved at runtime. I can note this in the docs though.